src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/OverlayList.js

```diff
@@ -41,7 +41,9 @@ export default class OverlayList extends React.PureComponent<Props> {
     @observable scrollHeight: number;
     @observable scrollWidth: number;
     @observable centeredChildRelativeTop: number;
+    @observable visible: boolean = false;
```

Comment:
`isOpen` for the sake of consistency?

Reply:
In other components this is called `isVisible`, shouldn't we be consistent here at least?